brightlayer-ui/react-native-workflows
# Verification code button style update

BLUI-3269
Fixes #157 


#### Changes proposed in this Pull Request:
- Verification code button style update


#### Screenshots:


#### To Test:
- Clone react-native-workflow project
- Initialize react-auth-shared submodule using branch `feature/3269-add-verification-code-prompt-translation-text`
- `yarn start:example`
- Go to self registration workflow (click debug button, then click self-registration button)
- Navigate to the third screen in the workflow to visit the verification code page
- Observe the updated view
